File: packages/cli/tests/commands/template/fixtures/start-cmd/expected/python-sync/template.py
from e2b import Template

template = (
    Template()
    .from_image("python:3.11")
    .set_user("root")
    .set_workdir("/")
    .set_workdir("/app")
    .run_cmd("pip install --upgrade pip")
    .run_cmd("pip install -r requirements.txt")
    .set_envs({
        "PYTHONUNBUFFERED": "1",
    })
    .set_user("user")
    .set_start_cmd("sudo node server.js", "sleep 20")
)

File: packages/cli/tests/commands/template/fixtures/start-cmd/expected/typescript/template.ts
import { Template } from 'e2b'

export const template = Template()
  .fromImage('python:3.11')
  .setUser('root')
  .setWorkdir('/')
  .setWorkdir('/app')
  .runCmd('pip install --upgrade pip')
  .runCmd('pip install -r requirements.txt')
  .setEnvs({
    'PYTHONUNBUFFERED': '1',
  })
  .setUser('user')
  .setStartCmd('sudo node server.js', 'sleep 20')